components/alert/alert.css

```diff
@@ -0,0 +1,39 @@
+.alert-box {
+  border-radius: 0.3rem;
+  padding: 1rem 3rem 1rem 1rem;
+  font-size: 1rem;
+  margin: 1rem 0;
+  position: relative;
+  width: 100%;
+  box-sizing: border-box;
```

Comment:
Good to have these things in the root file  so you don't need to write it for every class